trio/_ssl.py

```diff
@@ -159,53 +159,6 @@
 from . import _sync
 from ._util import ConflictDetector
 
-__all__ = ["SSLStream", "SSLListener"]
-
-################################################################
-# Faking the stdlib ssl API
-################################################################
-
-
-def _reexport(name):
-    try:
-        value = getattr(_stdlib_ssl, name)
-    except AttributeError:
-        pass
-    else:
-        globals()[name] = value
-        __all__.append(name)
-
-
-# Intentionally not re-exported:
-# SSLContext
-for _name in [
-    "SSLError",
-    "SSLZeroReturnError",
-    "SSLSyscallError",
-    "SSLEOFError",
-    "CertificateError",
-    "create_default_context",
-    "match_hostname",
-    "cert_time_to_seconds",
-    "DER_cert_to_PEM_cert",
-    "PEM_cert_to_DER_cert",
-    "get_default_verify_paths",
-    "Purpose",
-    "enum_certificates",
-    "enum_crls",
-    "SSLSession",
-    "VerifyMode",
-    "VerifyFlags",
-    "Options",
-    "AlertDescription",
-    "SSLErrorNumber",
-]:
-    _reexport(_name)
-
-for _name in _stdlib_ssl.__dict__.keys():
-    if _name == _name.upper():
-        _reexport(_name)
```

Comment:
This code is like `trio.socket`: it re-exports all the uppercased symbols. So we need some similar handling: a fake `try`/`import`, plus moving this loop into `trio/ssl.py`. With this PR right now it looks like we aren't re-exporting the uppercased symbols at all?

Reply:
They are now imported, but see my comment below about external dependencies like ssl library and version.